src/common/ContractInteractor.ts

```diff
@@ -217,7 +217,7 @@ export default class ContractInteractor {
   }
 
   async _validateCompatibility (): Promise<void> {
-    if (this.deployment == null) {
+    if (this.deployment == null || this.relayHubInstance == null) {
```

Comment:
how would you have an `instance` if the `deployment` is `null`? I think we initialize the instances from the deployment, don't we?

Reply:
added because it crashed.. we create ContractInteractor without any RH, but still call validateCompatibility on it. it used to pass because the "beta ignore" ignored also "can't find versionHub of undefined"